cmd/monaco/test-resources/integration-all-configs/project/reports/report.yaml

```diff
@@ -0,0 +1,6 @@
+config:
+  - report: "report.json"
+
+report:
+  - name: "/projecet/dashboard/dashboard.name"
```

Comment:
You have a typo here. `projecet` instead of `project`

Reply:
I guess you still have a typo here

Reply:
@patrickpichler correct it now